pkg/oc/cli/observe/observe.go

```diff
@@ -296,6 +302,10 @@ func (o *ObserveOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args [
 		return fmt.Errorf("you may only specify one argument containing the resource to observe (use '--' to separate your resource and your command)")
 	}
 
+	if _, err := labels.Parse(o.selector); err != nil {
```

Comment:
This belongs in `Validate` not `Complete`.

Reply:
Fixed by eba61e7.